cfme/tests/test_login.py

```diff
@@ -15,12 +16,28 @@
 @pytest.mark.smoke
 @pytest.mark.parametrize('context, method', [(ViaUI, 'click_on_login'),
                                              (ViaUI, 'press_enter_after_password'),
-                                             (ViaUI, '_js_auth_fn'),
-                                             (ViaSSUI, 'click_on_login'),
-                                             (ViaSSUI, 'press_enter_after_password')])
+                                             (ViaUI, '_js_auth_fn')])
 def test_login(context, method, appliance):
     """ Tests that the appliance can be logged into and shows dashboard page. """
+    with appliance.context.use(context):
+        logged_in_page = appliance.server.login()
+        assert logged_in_page.is_displayed
+        logged_in_page.logout()
+
+        logged_in_page = appliance.server.login_admin(method=method)
+        assert logged_in_page.is_displayed
+        logged_in_page.logout()
+
 
+@test_requirements.drift
+@pytest.mark.tier(1)
+@pytest.mark.sauce
+@pytest.mark.smoke
+@pytest.mark.parametrize('context, method', [(ViaSSUI, 'click_on_login'),
+                                             (ViaSSUI, 'press_enter_after_password')])
+@pytest.mark.uncollectif(lambda: current_version() > '5.8')
```

Comment:
Instead of copying the same test just to be able to uncollect it for SSUI, consider using smtg like
```python
@pytest.mark.uncollectif(lambda context: context == ViaSSUI and current_version() > '5.8')
```